src/main/resources/assets/botania/lang/en_us.json

```diff
@@ -2669,372 +2587,364 @@
-  "botania.page.thornChakram0": "The $(1)Thorn Chakram$(0) is a thrown weapon made of $(1)Terrasteel$(0) and plant matter. After thrown it'll pierce through enemies, damaging them and sometimes poisoning them, and bounce on blocks a few times before returning to the thrower, similarly to a boomerang.$(p)Up to 6 $(1)Thorn Chakrams$(0) can be stacked together, making them a versatile weapon for closed environments.",
+  "botania.page.thornChakram0": "The $(1)Thorn Chakram$(0) is a thrown weapon crafted from $(1)Terrasteel$(0) and plant matter. When thrown, a chakram pierces through enemies, dealing damage and having a chance to poison. It bounces off blocks, and after about three seconds returns to its thrower, as a boomerang does.$(p)Up to 6 $(1)Thorn Chakrams$(0) can be stacked together, making them deadly in closed spaces.",
   "botania.page.thornChakram1": "Spores and stingers",
 
   "botania.entry.fireChakram": "Flare Chakram",
   "botania.tagline.fireChakram": "A burning Thorn Chakram",
-  "botania.page.fireChakram0": "Some $(1)Blaze Powder$(0) and $(1)Pixie Dust$(0) does wonders when applied to a $(1)Thorn Chakram$(0), this combination turns the already potent weapon into a $(1)Flame Chakram$(0).$(p)The $(1)Flame Chakram$(0) works exactly the same as the original $(1)Thorn Chakram$(0), the only difference being that it sets any living beings it touches aflame, and after all, you can't make bacon with some just some plants' thorns, can you?",
+  "botania.page.fireChakram0": "$(1)Blaze Powder$(0) and $(1)Pixie Dust$(0), when applied to a $(1)Thorn Chakram$(0), turn the already potent weapon into a $(1)Flare Chakram$(0).$(p)The $(1)Flare Chakram$(0) works nearly the same as the original $(1)Thorn Chakram$(0), the only difference being its ability to set mobs aflame. After all, you can't make bacon from just vegetable matter, can you?",
   "botania.page.fireChakram1": "It's pretty lit",
 
   "botania.entry.overgrowthSeed": "Overgrowth Seed",
   "botania.tagline.overgrowthSeed": "An elusive object that powers up flowers",
-  "botania.page.overgrowthSeed0": "An $(1)Overgrowth Seed$(0) is an elusive item, which can be used to enchant a piece of $(1)Grass$(0), via right clicking on it in the world, creating $(1)Enchanted Soil$(0). Flowers that are rooted on this $(1)Enchanted Soil$(0) will function at double the normal speed.",
-  "botania.page.overgrowthSeed1": "There's one problem, though.$(p)No known method of creating these exists. They have however, been allegedly spotted by historians in ancient structures and temples. Perhaps that would be a wise guess as to where to find these.",
+  "botania.page.overgrowthSeed0": "An $(1)Overgrowth Seed$(0) is an elusive item that, when planted on $(1)Grass$(0), spells it into $(1)Enchanted Soil$(0). Flowers that are planted on this $(1)Enchanted Soil$(0) will function at double their normal speed-- rumor has it that its life-giving properties even prevent decay.",
```

Comment:
I can assure you that those life-giving properties are just a simple rumor. (It's been a good while since that was removed)